adding_species_manuscript.md

```diff
@@ -135,7 +117,7 @@ since most population-level summaries of genetic variation do not take into acco
 (However, this may change in the future.)
 The differences between one whole-chromosome simulation and many independent simulations of smaller pieces of genome
 are due to linkage, i.e., correlations in transmission and inheritance between adjacent parts of the genome.
-In any simulation, even neutral ones,
+In any simulation, even neutral ones,c
```

Comment:
Not a fan of this part of the commit. :-)

Reply:
Fair enough!